### ISSUE TITLE
Attach one Viro3DObject to an anchor on another Viro3DObject

### ISSUE THREAD

**Issue by adilosa** [viromedia/viro#651]
- [x] Review the documentation: https://docs.viromedia.com/
  - [x] Search for existing issues: https://github.com/viromedia/viro
  - [x] Use the latest ViroReact release: https://docs.viromedia.com/docs/releases

## Environment
Please provide the following information about your environment:
1. Development OS: Mac
2. Device OS & Version: iOS 12.2
3. Version: ViroReact 2.13.0 and React Native 0.57.8
4. Device(s): iPhone 8/iPhone X

## Description
I'm trying to build a character picker where the user can attach an accessory model (glasses/hat/bag/etc) to the base model. I can update the positions of the accessory to track the base model, but this won't follow the base model's animations. For example, glasses appear to float statically in front of my character's head, since the head bobs around in an idle animation.

Is it possible to "attach" one model to an anchor point on another model, as in a game engine?

**Comment by Turtleted21** [comment]
I don't think you can get a bone position. I think the best way is to bake your head character animation then apply this animation to a head  accessory :glasses, hat...
